ui/src/components/Nodes/NodesTable.vue

```diff
@@ -107,35 +157,107 @@ const sortChanged = (sortObj: FeatherSortObject) => {
   sort(sortObj)
 }
 
-const { queryParameters, updateQueryParameters, sort } = useQueryParameters({
-  limit: 10,
-  offset: 0,
-  orderBy: 'label'
-}, 'nodesModule/getNodes')
+const buildQueryParams = (searchVal: string) => {
```

Comment:
This function feels a bit long in the tooth. It feels copied from elsewhere, which is cool, but maybe just a breakdown into smaller chunks so it's self documenting (and easier to test) would be enough. Just a nit though, won't affect approval.

Reply:
This function is new, but will definitely be redone and broken down and refactored into smaller chunks and probably moved out of the Vue file into a library file, since we'll be adding even more filtering to it.